testbed.py

```diff
@@ -8,20 +8,29 @@
 
 
 def main():
-    attacker_builder = RandomAgentBuilder()
-    # attacker_builder = QCompatibilityAgentBuilder(
-    #     learner=torch.load('deepq.pkl')
-    # )
-    # attacker_builder = LoadFileBaselineAgentBuilder(A2C, 'ppo_marl_attacker.zip')
+    attackers = {
+        'random': RandomAgentBuilder(),
+        'tabularq': QCompatibilityAgentBuilder(learner=torch.load('tabularq.pkl')),
+        'deepq': QCompatibilityAgentBuilder(learner=torch.load('deepq.pkl')),
+        'ppo': LoadFileBaselineAgentBuilder(PPO, 'ppo.zip'),
+        'ppo_marl': LoadFileBaselineAgentBuilder(PPO, 'ppo_marl_attacker.zip'),
+        'a2c': LoadFileBaselineAgentBuilder(A2C, 'a2c.zip'),
+        'a2c_marl': LoadFileBaselineAgentBuilder(A2C, 'a2c_marl_attacker.zip'),
+    }
 
-    #defender_builder = LoadFileBaselineAgentBuilder(A2C, 'ppo_marl_defender.zip')
-    #defender_builder = RandomAgentBuilder()
-    defender_builder = None
+    defenders = {
+        'none': None,
+        'random': RandomAgentBuilder(),
+        'ppo': LoadFileBaselineAgentBuilder(PPO, 'ppo_defender.zip'),
+        'ppo_marl': LoadFileBaselineAgentBuilder(PPO, 'ppo_marl_defender.zip'),
+        'a2c': LoadFileBaselineAgentBuilder(A2C, 'a2c_defender.zip'),
+        'a2c_marl': LoadFileBaselineAgentBuilder(A2C, 'a2c_marl_defender.zip'),
+    }
 
     universe = MultiAgentUniverse.build(
-        attacker_builder=attacker_builder,
+        attacker_builder=attackers['random'],
         attacker_invalid_action_reward=0,
-        defender_builder=defender_builder,
+        defender_builder=defenders['none'],
```

Comment:
Could we automate the collection of the stats? If its not super simple then this looks good

Reply:
Yes, that's the evaluation script, separate file. Going to run it overnight assuming we finish training in time

Reply:
(so it could automatically run through x number of scenarios and spit out numbers at the end)